ts/sagas/contentLoaders.ts

```diff
@@ -69,19 +69,23 @@ export function* watchContentServiceLoadSaga(): Iterator<Effect> {
         typeof isFirstVisibleServiceLoadCompletedSelector
       > = yield select(isFirstVisibleServiceLoadCompletedSelector);
 
-      if (response.isRight() && response.value.status === 200) {
-        yield put(
-          contentServiceLoad.success({ serviceId, data: response.value.value })
-        );
-        // If the service is loaded for the first time (at first startup or when the
-        // cache is cleaned), the app shows the service list item without badge
-        if (!isFirstVisibleServiceLoadCompleted) {
-          yield put(markServiceAsRead(serviceId));
+      if (response.isRight()) {
+        if (response.value.status === 200 || response.value.status === 404) {
```

Comment:
Maybe an issue:
with this approach if a metadata is 404 it will be never request again (it is some)
To to avoid this we need to add another condition here 

from this
https://github.com/teamdigitale/io-app/blob/fdb39a95bd34a742bb553e413e9eb80007458d70/ts/sagas/startup/loadVisibleServicesHandler.ts#L126-L136
to this
```typescript
const serviceMetadataIdsToLoad = visibleServices
        .filter(service => {
          const serviceId = service.service_id;
          const storedMetadata = servicesMetadataById[serviceId];
          return (
            // Service load never occurs or it ends up with an error
            serviceContentIdsToLoad.indexOf(serviceId) === -1 &&
            !pot.isLoading(storedMetadata) &&
            (pot.isError(storedMetadata) ||
              pot.isNone(storedMetadata) ||
              (pot.isSome(storedMetadata) &&
                storedMetadata.value === undefined))
          );
        })
```
Just by adding this condition
```typescript
(pot.isSome(storedMetadata) &&
                storedMetadata.value === undefined)
```

Reply:
ok, updated within https://github.com/teamdigitale/io-app/pull/1426/commits/da53702c8b7716b6591af6a5464ed9dcaa332def!